tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -253,147 +253,84 @@ def FunctionExpression(function: str, *cols: Expression) -> Expression: ...
+    def row_type(self, fields: Union[Dict[str, DuckDBPyType], List[str]]) -> DuckDBPyType: ...
     def map_type(self, key: DuckDBPyType, value: DuckDBPyType) -> DuckDBPyType: ...
-    def __enter__(self) -> DuckDBPyConnection: ...
-    def __exit__(self, exc_type: object, exc: object, traceback: object) -> None: ...
-    @property
-    def description(self) -> Optional[List[Any]]: ...
-    @property
-    def rowcount(self) -> int: ...
+    def duplicate(self) -> DuckDBPyConnection: ...
+    def execute(self, query: object, parameters: object = None, multiple_parameter_sets: bool = False) -> DuckDBPyConnection: ...
+    def executemany(self, query: object, parameters: object = None) -> DuckDBPyConnection: ...
+    def close(self) -> None: ...
+    def interrupt(self) -> None: ...
+    def fetchone(self) -> Optional[tuple]: ...
+    def fetchmany(self, size: int = 1) -> List[Any]: ...
+    def fetchall(self) -> List[Any]: ...
+    def fetchnumpy(self) -> dict: ...
+    def fetchdf(self, **kwargs) -> pandas.DataFrame: ...
+    def fetch_df(self, **kwargs) -> pandas.DataFrame: ...
+    def df(self, **kwargs) -> pandas.DataFrame: ...
+    def fetch_df_chunk(self, vectors_per_chunk: int = 1, **kwargs) -> pandas.DataFrame: ...
+    def pl(self, rows_per_batch: int = 1000000) -> polars.DataFrame: ...
+    def fetch_arrow_table(self, rows_per_batch: int = 1000000) -> pyarrow.lib.Table: ...
+    def arrow(self, rows_per_batch: int = 1000000) -> pyarrow.lib.Table: ...
+    def fetch_record_batch(self, rows_per_batch: int = 1000000) -> pyarrow.lib.RecordBatchReader: ...
+    def torch(self) -> dict: ...
+    def tf(self) -> dict: ...
+    def begin(self) -> DuckDBPyConnection: ...
+    def commit(self) -> DuckDBPyConnection: ...
+    def rollback(self) -> DuckDBPyConnection: ...
+    def append(self, table_name: str, df: pandas.DataFrame, **kwargs) -> DuckDBPyConnection: ...
+    def register(self, view_name: str, python_object: object) -> DuckDBPyConnection: ...
+    def unregister(self, view_name: str) -> DuckDBPyConnection: ...
+    def table(self, table_name: str) -> DuckDBPyRelation: ...
+    def view(self, view_name: str) -> DuckDBPyRelation: ...
+    def values(self, values: List[Any]) -> DuckDBPyRelation: ...
+    def table_function(self, name: str, parameters: object = None) -> DuckDBPyRelation: ...
+    def read_json(self, name: str, **kwargs) -> DuckDBPyRelation: ...
+    def extract_statements(self, query: str) -> List[Statement]: ...
+    def sql(self, query: str, **kwargs) -> DuckDBPyRelation: ...
+    def query(self, query: str, **kwargs) -> DuckDBPyRelation: ...
+    def from_query(self, query: str, **kwargs) -> DuckDBPyRelation: ...
+    def read_csv(self, path_or_buffer: Union[str, StringIO, TextIOBase], **kwargs) -> DuckDBPyRelation: ...
+    def from_csv_auto(self, path_or_buffer: Union[str, StringIO, TextIOBase], **kwargs) -> DuckDBPyRelation: ...
```

Comment:
Can we expand the keyword arguments here (using `*, arg1=...`), similar to `read_parquet` in the same doc (see maybe also the old [numpy stubs](https://github.com/numpy/numpy-stubs/blob/master/numpy-stubs/__init__.pyi)).